packages/fast-tooling-react/src/css-editor/border/border.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { get } from "lodash-es";
+import Foundation, {
+    HandledProps,
+} from "@microsoft/fast-components-foundation-react";
+import { BorderStyleValue, CSSBorderHandledProps, CSSBorderProps, CSSBorderState, CSSBorderUnhandledProps } from "./border.props";
+
+export default class CSSBorder extends Foundation<
+    CSSBorderHandledProps,
+    CSSBorderUnhandledProps,
+    CSSBorderState
+> {
+
+    public static displayName: string = "CSSBorder";
+
+    protected handledProps: HandledProps<CSSBorderHandledProps> = {
+        data: void 0,
+        onChange: void 0,
+        managedClasses: void 0,
+    };
+
+    constructor(props: CSSBorderProps) {
+        super(props);
+
+        this.state = {
+            borderColor: get(this.props.data, "borderColor", ""),
+            borderStyle: get(this.props.data, "borderStyle", BorderStyleValue.solid),
+            borderWidth: get(this.props.data, "borderWidth", "")
+        };
+    }
+
+    public componentDidUpdate(prevProps: CSSBorderProps, prevState: CSSBorderState): void {
+        if (this.state !== prevState) {
+            this.props.onChange({
+                border: `${this.state.borderColor} ${this.state.borderStyle} ${this.state.borderWidth}`,
+            });
+        }
+    }
+
+    public render(): React.ReactNode {
+        return (
+            <div className={get(this.props, "managedClasses.cssBorder")}>
+                <label className={get(this.props, "managedClasses.cssBorder_label")}>
+                    Border
+                </label>
+                <div className={get(this.props, "managedClasses.cssBorder_control")}>
+                    <div className={get(this.props, "managedClasses.cssBorder_borderInputContainer")} style={{background: this.state.borderColor}}>
```

Comment:
should this be `cssBorder_colorInputRegion` like in the color component? It seems like it contains the input with type "color" so that naming makes sense to be repeated